recipes/aospy/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "aospy" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/27/39/d6674839265bbdccbd97291857baddf636e4329e03b6c179bcc62d390970/aospy-0.1.tar.gz
+  md5: 4201285d7171be95fa016039cceca719
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.7
```

Comment:
Sorry, but I'm not actually sure.  Here's a list of all the direct numpy calls we use:

abs
all
any
arange
array
concatenate
datetime64
deg2rad
diff
empty_like
float64
float_
int_
ma.add
ma.divide
ma.multiply
ma.power
ma.subtract
max
mean
min
ndarray
newaxis
ones
ones_like
pi
random.random
shape
sign
sin
squeeze
stack
testing.assert_allclose
testing.assert_array_equal
timedelta64
where
zeros